apps/dashboard/src/components/workflow-editor/steps/email/email-tabs.tsx

```diff
@@ -1,76 +1,41 @@
-import { Cross2Icon } from '@radix-ui/react-icons';
 import { useFormContext } from 'react-hook-form';
-import { RiEdit2Line, RiPencilRuler2Line } from 'react-icons/ri';
-import { useNavigate } from 'react-router-dom';
-
-import { Notification5Fill } from '@/components/icons';
-import { Button } from '@/components/primitives/button';
-import { Separator } from '@/components/primitives/separator';
-import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/primitives/tabs';
-import { StepEditorProps } from '@/components/workflow-editor/steps/configure-step-template-form';
+import { WorkflowOriginEnum } from '@novu/shared';
 import { EmailEditor } from '@/components/workflow-editor/steps/email/email-editor';
 import { EmailEditorPreview } from '@/components/workflow-editor/steps/email/email-editor-preview';
 import { CustomStepControls } from '../controls/custom-step-controls';
 import { EmailTabsSection } from '@/components/workflow-editor/steps/email/email-tabs-section';
-import { WorkflowOriginEnum } from '@novu/shared';
+import { StepEditorProps } from '@/components/workflow-editor/steps/configure-step-template-form';
+import { TemplateTabs } from '@/components/workflow-editor/steps/template-tabs';
 import { useState } from 'react';
 
-const tabsContentClassName = 'h-full w-full overflow-y-auto data-[state=inactive]:hidden';
-
 export const EmailTabs = (props: StepEditorProps) => {
   const { workflow, step } = props;
   const { dataSchema, uiSchema } = step.controls;
   const form = useFormContext();
-  const navigate = useNavigate();
   const [tabsValue, setTabsValue] = useState('editor');
 
-  return (
-    <Tabs defaultValue="editor" value={tabsValue} onValueChange={setTabsValue} className="flex h-full flex-1 flex-col">
-      <header className="flex flex-row items-center gap-3 px-3 py-1.5">
-        <div className="mr-auto flex items-center gap-2.5 text-sm font-medium">
-          <RiEdit2Line className="size-4" />
-          <span>Configure Template</span>
-        </div>
-        <TabsList className="w-min">
-          <TabsTrigger value="editor" className="gap-1.5">
-            <RiPencilRuler2Line className="size-5 p-0.5" />
-            <span>Editor</span>
-          </TabsTrigger>
-          <TabsTrigger value="preview" className="gap-1.5">
-            <Notification5Fill className="size-5 p-0.5" />
-            <span>Preview</span>
-          </TabsTrigger>
-        </TabsList>
+  const isNovuCloud = workflow.origin === WorkflowOriginEnum.NOVU_CLOUD && uiSchema;
```

Comment:
Our goal should be to align Dashboard and Code First workflow management. That is, uiSchema should be available in both. Let's remove it from this check.

Reply:
Currently its not available in both and the uiSchema type is optional on the response, there needs to be a check somewhere and this is the only place where the check makes sense.

If we want to achieve what you are describing there needs to be additional changes on API which I don't want to include as part of this UI bugfix.